CompfyuiOllama.py

```diff
@@ -39,6 +40,7 @@ def INPUT_TYPES(s):
                     "default": "http://127.0.0.1:11434"
                 }),
                 "model": ((), {}),
+                "seed": ("INT", {"default": seed, "min": 0, "max": 2 ** 31, "step": 1}),
```

Comment:
please put the seed value in the end of the list to not break existing workflows.

Reply:
Done.